src/main/java/org/example/CSVReader.java

```diff
@@ -0,0 +1,152 @@
+package org.example;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Time;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.example.Athlete.*;
+/* DESCRIPTION - Ski Biathlon Standings
+
+        write a software that takes as an input a CSV file where every entry represents the results of a biathlon athlete
+        based on the entries identify the first 3 places (Winner, Runner-up and Third Place)
+        the rules are simple, each athlete has a time-results for the entire skiing session, and 3 shooting range results
+        each shooting range has 5 shot results
+        for every missed shot the athlete obtains a 10 second penalty which affects the final time-result
+final standings are based on the time-results that have been updated with the shooting range results
+        CSV example
+
+        11,Umar Jorgson,SK,30:27,xxxox,xxxxx,xxoxo
+
+        1,Jimmy Smiles,UK,29:15,xxoox,xooxo,xxxxo
+
+        27,Piotr Smitzer,CZ,30:10,xxxxx,xxxxx,xxxxx
+
+        COLUMNS
+
+        AthleteNumber,AthleteName,CountryCode,SkiTimeResult(Minutes:Seconds),FirstShootingRange.SecondShooting,ThirdShooting
+
+        every shooting range consists of 5 characters, where x means hit, o mean miss
+        for every o there should be 10 seconds added to the final time
+        RESULTS
+
+        Winner - Piotr Smitzer 30:10 (30:10 + 0)
+
+        Runner-up - Jimmy Smiles 30:15 (29:15 + 60)
+
+        Third Place - Umar Jorgson 30:57 (30:27 + 30)
+
+        TASK
+
+        write tests for the CSV parsing and the standing calculation
+
+        in the your tests you must not use real files
+
+        make sure the CSVs are read from memory to keep the tests fast
+
+        use Comparator / Comparable for sorting
+
+        use at least 1 Enum (in any situation) and at least 2 Annotations (they don't need to be custom)*/
+
+public class CSVReader {
+
+
+    public static Athlete readFromCSV(){
+
+        // Read from CSV
+        List<String> rows= new ArrayList<>();
+        try {
+            BufferedReader bf = new BufferedReader( new FileReader("file.csv"));
+
+            String line;
+            while(bf.readLine()!=null) {
+                line = bf.readLine();
+                rows.add(line);
```

Comment:
spit into several methods, don't put all the work into a single method. 1 method = 1 action
readCsv
createAthlete
calculatePenalties (or total time)
sortAthletes
printResults